pkgs/development/php-packages/decimal/default.nix

```diff
@@ -0,0 +1,28 @@
+{
+  buildPecl,
+  lib,
+  mpdecimal,
+  php,
+}:
+let
+  version = "1.5.0";
+in
+buildPecl {
+  pname = "decimal";
+
+  version = version;
+  sha256 = "it8w8hOLYwtCZoDYhaP5k5TD/pQLtj37K2lSESF80ok=";
+
+  internalDeps = lib.optional (lib.versionOlder php.version "8.0") php.extensions.json;
```

Comment:
In Nixpkgs, we avoid provisions for versions of PHP not supported in Nixpkgs since the would not be testable.

```suggestion
```

We do that in nix-phps, for example: https://github.com/fossar/nix-phps/blob/224db3b3e29451853fd403a389ac35615427d61e/pkgs/package-overrides.nix#L89-L97